overturemaps/core.py

```diff
@@ -14,11 +14,14 @@
     HAS_GEOPANDAS = False
     GeoDataFrame = None
 
-def record_batch_reader(overture_type, bbox=None) -> Optional[pa.RecordBatchReader]:
+def record_batch_reader(overture_type, bbox=None, release=None) -> Optional[pa.RecordBatchReader]:
     """
     Return a pyarrow RecordBatchReader for the desired bounding box and s3 path
     """
-    path = _dataset_path(overture_type)
+
+    if release is None:
+        release = '2025-06-25.0'
```

Comment:
Can you make this value a constant at the top of the file. Also use it for the default value in the cli.